edgebuilder-install.sh

```diff
@@ -1,44 +1,21 @@
 #!/bin/sh
-COMPONENT=$1
-shift
 
 # Displays simple usage prompt
 display_usage()
 {
-  echo "Usage: edgebuilder-install.sh [param]"
+  echo "Usage: edgebuilder-install.sh [param] [options]"
   echo "params: server, node, cli"
+  echo "options: "
+  echo "     -r, --repo-auth : IoTech repo auth token to access packages"
+  echo "     -u              : Uninstall the package"
+  echo "     -f, --file      : path to local package"
```

Comment:
I can understand why we might want to move the while statement below all functions, but can we move the display_usage() to just above the while loop? I ask because it's then closer to where it's used, it's easier to reference in code and it's only really used in the while loop.

Reply:
yep sure, it might also make the code's API easier to understand